.circleci/config.yml

```diff
@@ -10,10 +29,30 @@ jobs:
         - checkout
 
         - run:
-            name: Install Serverless CLI and dependencies
+            name: Install dependencies
+            command: npm install
+        - run:
+            name: Deploy application
             command: |
-              npm install
+              if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
I'm not sure about the yml syntax but see if you can combine this with the if statement above (if... else...)

Reply:
Change looks great